src/storage/settings/settingsSlice.ts

```diff
@@ -46,15 +47,22 @@ import SettingsStorage from '@/storage/settings/settingsPersistentStorage'
 import { RootState } from '@/storage/store'
 import { GeneralSettings } from '@/types/settings'
 
-const initialState: GeneralSettings = SettingsStorage.load('general') as GeneralSettings
+interface SettingsState extends GeneralSettings {
+  faucetCallPending: boolean
```

Comment:
Tbh, not a big fan of using this slice for storing this. I think it would fit better the `global/globalSlice.tsx`. WDYT?

Reply:
Good idea!